### PR TITLE
changing keyboard mapping

### PR DESCRIPTION
**Pull request by druiz17**
changing keyboard mapping

Ticket - https://highfidelity.manuscript.com/f/cases/12498/Update-control-scheme-for-keyboard-navigation

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7781/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7781/HighFidelity-Beta-PR12574-6fefd1355aa53dc0fb545cb8148ac151740102f8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7781/HighFidelity-Beta-PR12574-6fefd1355aa53dc0fb545cb8148ac151740102f8.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7793/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7793/HighFidelity-Beta-PR12574-69af12fbc7648ab1de36034907b40c986dd02105.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7793/HighFidelity-Beta-PR12574-69af12fbc7648ab1de36034907b40c986dd02105.exe

**Comment by alexiamandeville** [comment]
I think the shift modifier on walk could be a bit faster, but overall looks good.

**Comment by druiz17** [comment]
Test plan
- open the PDF below and perform the actions highlight in green make sure they do what the action says
- the red highlighted actions have been removed, make sure you can no longer perform them
- Also perform the actions for  `Strafe Left` and Strafe Right`
[KeyboardControls_update.pdf](https://github.com/highfidelity/hifi/files/1789933/KeyboardControls_update.pdf)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7815/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7815/HighFidelity-Beta-PR12574-a70509233e22bb3e70748d3716601e99a808c30a.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7818/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7818/HighFidelity-Beta-PR12574-a70509233e22bb3e70748d3716601e99a808c30a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7844/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7844/HighFidelity-Beta-PR12574-21781c0c40dc170446bb9b29211fc4f68b6b1ba7.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7878/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7878/HighFidelity-Beta-PR12574-21781c0c40dc170446bb9b29211fc4f68b6b1ba7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7878/HighFidelity-Beta-PR12574-21781c0c40dc170446bb9b29211fc4f68b6b1ba7.exe

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -352,7 +352,7 @@ static const QString AVA_JSON_EXTENSION = ".ava.json";
 static const QString WEB_VIEW_TAG = "noDownload=true";
 static const QString ZIP_EXTENSION = ".zip";
 
-static const float MIRROR_FULLSCREEN_DISTANCE = 0.389f;
+static const float MIRROR_FULLSCREEN_DISTANCE = 0.789f;
```
Comment (sethalves) [inline interface/src/Application.cpp]: why was this changed?

**Comment by qlimnavarro** [comment]
Currently Testing.

**Comment by qlimnavarro** [comment]
Pass @conklin94122

File: interface/src/Application.cpp (hunk 1 above)
Comment (alexiamandeville) [reply interface/src/Application.cpp]: Design proposal for wider FOV in mirror mode provided by me. I obv didn't make the change but I assume this is what it's for.

**Comment by conklin94122** [comment]
@druiz17 Needs dev review

File: interface/src/Menu.cpp [hunk 2]
```diff
@@ -225,21 +225,21 @@ Menu::Menu() {
 
     // View > First Person
     auto firstPersonAction = cameraModeGroup->addAction(addCheckableActionToQMenuAndActionHash(
-                                   viewMenu, MenuOption::FirstPerson, Qt::CTRL | Qt::Key_F,
+                                   viewMenu, MenuOption::FirstPerson, Qt::SHIFT | Qt::Key_F,
```
Comment (ZappoMan) [inline interface/src/Menu.cpp]: this is a violation of the windows and mac OS design guidelines.... all menu item "hot keys" need to use a modifier. Having a menu item hot key directly mapped to a non-modified key is very non-standard. Are we ABSOLUTELY certain we want this. @alexiamandeville

File: interface/src/Menu.cpp [hunk 3]
```diff
@@ -225,21 +225,21 @@ Menu::Menu() {
...
                                    true, qApp, SLOT(cameraMenuChanged())));
 
     firstPersonAction->setProperty(EXCLUSION_GROUP_KEY, QVariant::fromValue(cameraModeGroup));
 
     // View > Third Person
     auto thirdPersonAction = cameraModeGroup->addAction(addCheckableActionToQMenuAndActionHash(
-                                   viewMenu, MenuOption::ThirdPerson, Qt::CTRL | Qt::Key_G,
+                                   viewMenu, MenuOption::ThirdPerson, Qt::SHIFT | Qt::Key_G,
```
Comment (ZappoMan) [inline interface/src/Menu.cpp]: this is a violation of the windows and mac OS design guidelines.... all menu item "hot keys" need to use a modifier. Having a menu item hot key directly mapped to a non-modified key is very non-standard. Are we ABSOLUTELY certain we want this. @alexiamandeville

File: interface/src/Menu.cpp [hunk 4]
```diff
@@ -225,21 +225,21 @@ Menu::Menu() {
...
                                    false, qApp, SLOT(cameraMenuChanged())));
 
     thirdPersonAction->setProperty(EXCLUSION_GROUP_KEY, QVariant::fromValue(cameraModeGroup));
 
     // View > Mirror
     auto viewMirrorAction = cameraModeGroup->addAction(addCheckableActionToQMenuAndActionHash(
-                                   viewMenu, MenuOption::FullscreenMirror, Qt::CTRL | Qt::Key_H,
+                                   viewMenu, MenuOption::FullscreenMirror, Qt::SHIFT | Qt::Key_H,
```
Comment (ZappoMan) [inline interface/src/Menu.cpp]: this is a violation of the windows and mac OS design guidelines.... all menu item "hot keys" need to use a modifier. Having a menu item hot key directly mapped to a non-modified key is very non-standard. Are we ABSOLUTELY certain we want this. @alexiamandeville

File: interface/src/avatar/MyAvatar.cpp [hunk 5]
```diff
@@ -2189,7 +2190,8 @@ void MyAvatar::updateActionMotor(float deltaTime) {
     if (state == CharacterController::State::Hover ||
             _characterController.computeCollisionGroup() == BULLET_COLLISION_GROUP_COLLISIONLESS) {
         // we can fly --> support vertical motion
-        glm::vec3 up = (getDriveKey(TRANSLATE_Y)) * IDENTITY_UP;
+        float impulse = _sprint ? 4.0f : 1.0f;
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.cpp]: coding standard: magic numbers. 4.0 and 1.0 should be named constants `SPRINT_IMPULSE` and `WALK_IMPULSE`

File: interface/src/avatar/MyAvatar.cpp [hunk 6]
```diff
@@ -2209,7 +2211,7 @@ void MyAvatar::updateActionMotor(float deltaTime) {
         float motorSpeed = glm::length(_actionMotorVelocity);
         float finalMaxMotorSpeed = getSensorToWorldScale() * DEFAULT_AVATAR_MAX_FLYING_SPEED;
         float speedGrowthTimescale  = 2.0f;
-        float speedIncreaseFactor = 1.8f;
+        float speedIncreaseFactor = _sprint ? 5.0f : 1.8f;
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.cpp]: coding standard: magic numbers, should be named constants

File: interface/src/avatar/MyAvatar.cpp [hunk 7]
```diff
@@ -2821,6 +2823,12 @@ float MyAvatar::getWalkSpeed() const {
     return _walkSpeed.get();
 }
 
+void MyAvatar::setSprintMode(bool sprint) {
+    _sprint = sprint;
+    float sprintSpeed = _sprint ? AVATAR_RUN_SPEED : DEFAULT_AVATAR_MAX_WALKING_SPEED;
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.cpp]: this looks weird to be on 2 lines and to have an extra local variable named `sprintSpeed`

I seems like this would be better:
`_walkSpeed.set(_sprint ? AVATAR_RUN_SPEED : DEFAULT_AVATAR_MAX_WALKING_SPEED);`
or split it up with an if statement...
```
if (_sprint) {
    _walkSpeed.set(AVATAR_RUN_SPEED);
} else {
    _walkSpeed.set(DEFAULT_AVATAR_MAX_WALKING_SPEED);
}
```

File: interface/src/avatar/MyAvatar.h [hunk 8]
```diff
@@ -839,6 +840,7 @@ private slots:
     mutable std::mutex _controllerPoseMapMutex;
 
     bool _hmdLeanRecenterEnabled = true;
+    bool _sprint = false;
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.h]: coding standard says: `bool _sprint { false };

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7932/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7932/HighFidelity-Beta-PR12574-c949b44f0c8139b64d5ad1e7623a703cf0b7f657.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7932/HighFidelity-Beta-PR12574-c949b44f0c8139b64d5ad1e7623a703cf0b7f657.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7972/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7972/HighFidelity-Beta-PR12574-272f71b940786cdea6ec911f256563617782f51b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7972/HighFidelity-Beta-PR12574-272f71b940786cdea6ec911f256563617782f51b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7977/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7978/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7978/HighFidelity-Beta-PR12574-e289d37210bc41865a7ec734fe341c7aaa188d98.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7978/HighFidelity-Beta-PR12574-e289d37210bc41865a7ec734fe341c7aaa188d98.exe

**Comment by alexiamandeville** [comment]
Issues:
1. Shortcuts no longer work - navigation is overriding them. (CTRL+J) It's fine if you move a little when using a shortcut since they are few.
2. When sprinting, A and D become strafe. They should still be rotate, since we have strafe on Q and E.

**Comment by alexiamandeville** [comment]
User should not be able to walk/move in Mirror mode.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7997/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7997/HighFidelity-Beta-PR12574-00fa5cfd0632aca13e130c69bc0c9434ab192ebf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7997/HighFidelity-Beta-PR12574-00fa5cfd0632aca13e130c69bc0c9434ab192ebf.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8014/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8014/HighFidelity-Beta-PR12574-f9413dfe1b5780eb8e6edc9b3fe1dace4ac8d6c7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8014/HighFidelity-Beta-PR12574-f9413dfe1b5780eb8e6edc9b3fe1dace4ac8d6c7.exe

**Comment by conklin94122** [comment]
@druiz17 I'm going to remove QA Ready until the dev review passes. Please add it back on after that, we'll test it ASAP.

**Comment by qlimnavarro** [comment]
Currently testing.

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
Ctrl + Shift + A for asset browser does not work. 

While sprinting A and D now turn the avatar rather than strafe. No longer able to move in Mirror mode.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8111/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8111/HighFidelity-Beta-PR12574-1dcdcffa69891920ab109d41788f3c6fe8c3abf2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8111/HighFidelity-Beta-PR12574-1dcdcffa69891920ab109d41788f3c6fe8c3abf2.exe

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
On Mac OS, the interface crashes after about 5 minutes. Logs and video attached, this does **not** occur on Master (7982) or Stable (7977).
[Files.zip](https://github.com/highfidelity/hifi/files/1825679/Files.zip)

File: interface/src/avatar/MyAvatar.cpp [hunk 9]
```diff
@@ -2819,6 +2825,16 @@ float MyAvatar::getWalkSpeed() const {
...
 
+void MyAvatar::setSprintMode(bool sprint) {
+    _sprint = sprint;
+
+    if (_sprint) {
+        _walkSpeed.set(AVATAR_RUN_SPEED);
+    } else {
+        _walkSpeed.set(DEFAULT_AVATAR_MAX_WALKING_SPEED);
+    }
+}
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyAvatar.cpp]: `MyAvatar._walkSpeed` is used as a "property" of the `MyAvatar` JS object.  This new `setSprintMode()` method would overwrite any changes that were made by a JS script.

Maybe setting the sprint mode should instead set a multiplicative factor (1.0 for "walk" or 3.0 for "sprint") that is used to scale `_walkSpeed` the "finalScaledWalkSpeed" (or whatever it is called) right before it is used?

File: interface/src/avatar/MyAvatar.cpp [hunk 10]
```diff
@@ -71,6 +71,7 @@ const float YAW_SPEED_DEFAULT = 100.0f;   // degrees/sec
 const float PITCH_SPEED_DEFAULT = 75.0f; // degrees/sec
 
 const float MAX_BOOST_SPEED = 0.5f * DEFAULT_AVATAR_MAX_WALKING_SPEED; // action motor gets additive boost below this speed
+const float AVATAR_RUN_SPEED = DEFAULT_AVATAR_MAX_WALKING_SPEED  * 3.0f;
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyAvatar.cpp]: This should probably be called `DEFAULT_AVATAR_RUN_SPEED`.  The "DEFAULT" modifier is supposed to remind us devs that this is the speed for a "default" sized avatar.  If the user were to scale their avatar up or down then this value would be scaled accordingly before it is applied.

Hrm... looking at this code it appears that maybe the avatar scale is not yet being taken into account.  I'll have to audit the full code to be sure.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8175/app_12574_PR.apk

File: interface/src/avatar/MyAvatar.cpp [hunk 11]
```diff
@@ -2187,7 +2188,10 @@ void MyAvatar::updateActionMotor(float deltaTime) {
     if (state == CharacterController::State::Hover ||
             _characterController.computeCollisionGroup() == BULLET_COLLISION_GROUP_COLLISIONLESS) {
         // we can fly --> support vertical motion
-        glm::vec3 up = (getDriveKey(TRANSLATE_Y)) * IDENTITY_UP;
+        const float SPRINT_IMPULSE = 4.0f;
+        const float WALK_IMPULSE = 1.0f;
+        float impulse = _sprint ? SPRINT_IMPULSE : WALK_IMPULSE;
+        glm::vec3 up = (getDriveKey(TRANSLATE_Y) * impulse) * IDENTITY_UP;
         direction += up;
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyAvatar.cpp]: I don't think these changes are doing what you want.  At this stage we're just trying to figure out what is the net `direction` of the hover.  The `direction` will be normalized (or zeroed as the case may be) further down.  By only adjusting the `up` component of `direction` for sprint (and not the `forward` and `right` components) you would distort the hover to be more `up` than `horizontal` during sprint.

My advice would be to roll this section back to the original state and instead rely on speeding up the hover speed growth (or maybe just scaling the final hover speed) later.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8175/HighFidelity-Beta-PR12574-cce77e3e68c468cb67894c5d6883635a9872d012.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8175/HighFidelity-Beta-PR12574-cce77e3e68c468cb67894c5d6883635a9872d012.exe

File: interface/src/avatar/MyAvatar.cpp [hunk 12]
```diff
@@ -2204,10 +2208,12 @@ void MyAvatar::updateActionMotor(float deltaTime) {
...
         // we're flying --> complex acceleration curve that builds on top of current motor speed and caps at some max speed
+        const float WALK_SPEED_FACTOR = 1.8f;
+        const float SPRINT_SPEED_FACTOR = 5.0f;
         float motorSpeed = glm::length(_actionMotorVelocity);
         float finalMaxMotorSpeed = getSensorToWorldScale() * DEFAULT_AVATAR_MAX_FLYING_SPEED;
         float speedGrowthTimescale  = 2.0f;
-        float speedIncreaseFactor = 1.8f;
+        float speedIncreaseFactor = _sprint ? SPRINT_SPEED_FACTOR : WALK_SPEED_FACTOR;
         motorSpeed *= 1.0f + glm::clamp(deltaTime / speedGrowthTimescale, 0.0f, 1.0f) * speedIncreaseFactor;
         const float maxBoostSpeed = getSensorToWorldScale() * MAX_BOOST_SPEED;
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyAvatar.cpp]: The hover speed ramps up from slow to fast.  These changes make it ramp up faster when in "sprint" mode, but don't actually change the max hover speed.  I guess that is correct, although just using a linear multiplier to the final speed (and thus max hover speed) would be another way to do it.

File: interface/src/avatar/MyAvatar.cpp (hunk 10 above)
Comment (AndrewMeadows) [reply interface/src/avatar/MyAvatar.cpp]: After looking at the code I see that `MyAvatar::_walkSpeed` is scaled appropriately right before it is passed to the "motors" which are used in the physics simulation.

File: interface/src/Application.cpp [hunk 13]
```diff
@@ -2812,12 +2813,15 @@ void Application::updateCamera(RenderArgs& renderArgs, float deltaTime) {
                 + mirrorBodyOrientation * hmdOffset);
         }
         else {
-            _myCamera.setOrientation(myAvatar->getWorldOrientation()
-                * glm::quat(glm::vec3(0.0f, PI + _rotateMirror, 0.0f)));
+            auto userInputMapper = DependencyManager::get<UserInputMapper>();
+            const float YAW_SPEED = TWO_PI / 10.0f;
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: @druiz17, sorry the `TWO_PI / 10.0f` value was just an example rate where I was trying to convey a good way to write it.   The main benefit being it is easy to _understand_ what it is: "one revolution every 10 seconds".  The final correct number of seconds/revolution probably needs to be _tuned_ to agree with what a user would expect from previous behavior.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8183/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8183/HighFidelity-Beta-PR12574-9d8770b1346e4e66426ee844b1b32fe1f563db61.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8183/HighFidelity-Beta-PR12574-9d8770b1346e4e66426ee844b1b32fe1f563db61.exe

**Comment by conklin94122** [comment]
Relabel as QA Ready when changes are applied.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8230/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8231/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8231/HighFidelity-Beta-PR12574-f36a916fb686a77bf36718901151fba048400dcf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8231/HighFidelity-Beta-PR12574-f36a916fb686a77bf36718901151fba048400dcf.exe

**Comment by RyanDowne** [comment]
can this be retested now?

**Comment by druiz17** [comment]
I want Alexia to confirm the new changes before it goes through QA.

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
The following key binding dont seem to work as expected:
1) Alternate for strafe should be Shift + Left/Right Arrow key. Instead it just turns the avatar.
2) While in mirror mode the user should not be able to use the arrow keys to look around. In mirror mode currently the camera moves with arrow keys.

Notes: Since Ctrl is now a movement command it does make the UI disappear for a second, keybaord shortcuts still work as expected. The new speed is well recieved fast enough to cover long distances. Controls feel intuitive and good.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8240/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8241/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8241/HighFidelity-Beta-PR12574-28e61bcbb12c03ba9d015769f887bd044aca5e34.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Fail @conklin94122
Shift + A and D do nothing. in previous versions the avatar would continue to sprint and turn. Now while sprinting the keys do nothing.

Note: While sprinting, the Left/Right arrow keys do now strafe but they lose all speed when compared to Q and E.

[hifi-log_2018-03-23_14.20.35_4882865b-7832-456e-8bd9-341c7b2fddcd.txt](https://github.com/highfidelity/hifi/files/1843387/hifi-log_2018-03-23_14.20.35_4882865b-7832-456e-8bd9-341c7b2fddcd.txt)
[hifi-log.txt](https://github.com/highfidelity/hifi/files/1843388/hifi-log.txt)

**Comment by druiz17** [comment]
The sprint key will not work when performing the shift+left or shift+right actions

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8248/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8248/HighFidelity-Beta-PR12574-96707a256818d390fd52916dac0e83ce832a1212.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8248/HighFidelity-Beta-PR12574-96707a256818d390fd52916dac0e83ce832a1212.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Fail @conklin94122 
Shift + Up/Down arrow move the camera up and down. These controls were highlighted in red on the document provided. In addition, in HMD the Shift + Left/Right Arrow keys do not strafe, they instead turn the avatar. As a side note, on Mac the controls for print screen are Command+Shift+3, if the user misses the Shift key, the 3D TV view will fill up the whole screen, confusing the user.

**Comment by conklin94122** [comment]
Moving to RC67, ok'd by Dante.

**Comment by druiz17** [comment]
@qlimnavarro The action for snapshot is no longer `Print Screen` it is just the `p` key

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8256/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8256/HighFidelity-Beta-PR12574-1f23e7ab8b597bef37ecfd7e7ba4b930476212c7.exe

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8281/app_12574_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8281/HighFidelity-Beta-PR12574-1f23e7ab8b597bef37ecfd7e7ba4b930476212c7.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8281/HighFidelity-Beta-PR12574-1f23e7ab8b597bef37ecfd7e7ba4b930476212c7.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Keys now match the document and feel and intuitive.

**Comment by conklin94122** [comment]
@sethalves Needs techlead review
